### PR TITLE
updated Thala TVL logic

### PR DESCRIPTION
**Pull request by LawsonGraham**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### methodology (what is being counted as tvl, how is tvl being calculated):
 We've decided to include THL in our TVL calculations as this seems to be industry standard (for instance, we [see here](https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/helper/getUniSubgraphTvl.js) Uniswap does this with the UNI Token)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/move-dollar exports TVL: 
        
 
 
    aptos                     7.49 M
    
    total                    7.49 M

**Comment by llamatester** [comment]
The adapter at projects/thalaswap exports TVL: 
        
 
 
    aptos                     6.75 M
    
    total                    6.75 M

File: projects/thalaswap/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = {
         assets.forEach((asset, index) => {
           // We ignore the null coin because it signifies that we don't have either a third or fourth asset in the pool
           // We  ignore the THL coin because native tokens are exempt from TVL calculations
-          if (!asset_types[index].includes(nullCoinType) && !asset_types[index].includes(thlCoinType))
```
Comment (0xbe1) [inline projects/thalaswap/index.js:34]: can also remove line 7

**Review by 0xbe1 — COMMENTED** [review]
would be great to add explanation in PR why we want to do the update

**Comment by llamatester** [comment]
The adapter at projects/move-dollar exports TVL: 
        
 
 
    aptos                     7.48 M
    
    total                    7.48 M

**Comment by llamatester** [comment]
The adapter at projects/thalaswap exports TVL: 
        
 
 
    aptos                     6.75 M
    
    total                    6.75 M

File: projects/move-dollar/index.js [hunk 2]
```diff
@@ -29,6 +32,14 @@ module.exports = {
           sdk.util.sumSingleBalance(balances, asset_type, total_collateral);
       });
 
+      const stabilityPools = resources.filter(stabilityPoolsFilter).map(pool => ({
+          total_value: pool.data.stability.value,
+        }));
+      
+      stabilityPools.forEach(({ total_value }) => {
+          sdk.util.sumSingleBalance(balances, MODType, total_value);
```
Comment (g1nt0ki) [inline projects/move-dollar/index.js:40]: Sorry, we dont include own stable tokens in stable pools as tvl, we apply the same rule for all CDPs, the reasoning for this is, backing of the stabletokens are already counted towards protocol tvl

**Comment by llamatester** [comment]
Error while running adapter at projects/move-dollar: 
        
 
  
    
    
    ReferenceError: stabilityPoolsFilter is not defined
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/move-dollar/index.js:34:47)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:148:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:193:3

**Comment by llamatester** [comment]
The adapter at projects/move-dollar exports TVL: 
        
 
 
    aptos                     9.15 M
    
    total                    9.15 M

**Comment by llamatester** [comment]
The adapter at projects/thalaswap exports TVL: 
        
 
 
    aptos                     10.66 M
    
    total                    10.66 M

File: projects/move-dollar/index.js (hunk 2 above)
Comment (LawsonGraham) [reply projects/move-dollar/index.js]: okay, sounds good - updated now!

**Pull request merged**